system/filesystem.py

```diff
@@ -41,6 +41,12 @@
     description:
     - If yes, allows to create new filesystem on devices that already has filesystem.
     required: false
+  growfs:
+    choices: [ "yes", "no" ]
+    default: "no"
+    description:
+    - If yes, if the block device and filessytem size differ, grow the filesystem into the space. This DOES NOT support shrinking. Will only grow if the FS is mounted.
```

Comment:
Ansible usage is not limited to playbooks. It is often used as orchestration engine, for massive actions on remote hosts. For me, shrinking makes sense in orchestration.

As for comment about mounted FS, we could change it to mention XFS, or (better) add check to module, so running ansible in check mode could warn us about not mounted XFS.